app/models/opentofu_worker.rb

```diff
@@ -82,6 +82,9 @@ def configure_service_worker_deployment(definition)
     env_var_array = definition[:spec][:template][:spec][:containers][0][:env]
     env_var_array.detect { |env| env[:name] == "HOME" }&.[]=(:value, "/home/node")
 
+    definition[:spec][:template][:spec][:containers][0][:env] << {:name => "LOG4JS_LEVEL", :value => "info"}
+    definition[:spec][:template][:spec][:containers][0][:env] << {:name => "AIRGAP_OFFLINE", :value => "false"}
```

Comment:
Should these be driven by settings or something? If they're going to be static values, why set them here and not in the image?

Reply:
Customer can change based on requirement. e.g. 
- if we want more logs during issue debugging we can ask customer to change LOG4JS_LEVEL to 'debug'
- if customer is running on airgap environment, they have to change 'AIRGAP_OFFLINE' value to true

Reply:
What do you mean by setting? Can we have such setting which can tell terraform runner deployment that customer is running on airgap environment ?

Reply:
Every worker has a settings section, e.g. https://github.com/ManageIQ/manageiq-providers-embedded_terraform/blob/master/config/settings.yml#L19-L21

E.g. take a look at how we're handling the `:container_image` setting